torch/csrc/mps/Module.cpp

```diff
@@ -224,6 +224,22 @@ static PyObject* MPSModule_elapsedTimeOfEvents(
   END_HANDLE_TH_ERRORS
 }
 
+static PyObject* MPSModule_getCommandBuffer_raw(
+    PyObject* /* unused */,
+    PyObject* /* noargs, unused */) {
+  HANDLE_TH_ERRORS
+  return PyLong_FromVoidPtr(at::detail::getMPSHooks().getCommandBuffer());
```

Comment:
Do we care about the lifetime of the object here? I.e. should you increment the reference to it and return some sort of pythonic object?